code/modules/antagonists/incursion/incursion.dm

```diff
@@ -0,0 +1,182 @@
+	owner.current.playsound_local(get_turf(owner.current), 'sound/ambience/antag/tatoralert.ogg', 100, FALSE, pressure_affected = FALSE)
+	equip()
+
+/datum/antagonist/incursion/admin_add(datum/mind/new_owner,mob/admin)
+	//show list of possible brothers
+	var/list/candidates = list()
+	for(var/mob/living/L in GLOB.alive_mob_list)
+		if(!L.mind || L.mind == new_owner || !can_be_owned(L.mind))
+			continue
+		candidates[L.mind.name] = L.mind
+
+	if(SSticker.mode.incursion_team)
+		new_owner.add_antag_datum(/datum/antagonist/incursion, SSticker.mode.incursion_team)
+	else
+		SSticker.mode.incursion_team = new
+		SSticker.mode.incursion_team.add_member(new_owner)
+		SSticker.mode.incursion_team.forge_team_objectives()
+		new_owner.add_antag_datum(/datum/antagonist/incursion, SSticker.mode.incursion_team)
+		message_admins("New incursion team created by [key_name_admin(admin)]")
+	message_admins("[key_name_admin(admin)] made [key_name_admin(new_owner)] and [key_name_admin(new_owner.current)] into blood brothers.")
+	log_admin("[key_name(admin)] made [key_name(new_owner)] and [key_name(new_owner.current)] into incursion traitor team.")
+
+/datum/antagonist/incursion/proc/equip(var/silent = FALSE)
+	owner.equip_traitor("The Syndicate", silent, src)
+	var/obj/item/implant/radio/syndicate/syndio = new
+	syndio.implant(owner.current)
+
+/datum/team/incursion
+	name = "syndicate incursion force"
+	member_name = "incursion member"
+
+/datum/team/incursion/is_solo()
+	return FALSE
+
+/datum/team/incursion/roundend_report()
+	var/list/parts = list()
+
+	parts += "<span class='header'>The members of the Syndicate incursion were:</span>"
+	for(var/datum/mind/M in members)
+		parts += printplayer(M)
+	var/win = TRUE
+	var/objective_count = 1
+	for(var/datum/objective/objective in objectives)
+		if(objective.check_completion())
+			parts += "<B>Objective #[objective_count]</B>: [objective.explanation_text] <span class='greentext'>Success!</span>"
+		else
+			parts += "<B>Objective #[objective_count]</B>: [objective.explanation_text] <span class='redtext'>Fail.</span>"
+			win = FALSE
+		objective_count++
+	if(win)
+		parts += "<span class='greentext'>The Syndicate were successful with their operation!</span>"
+	else
+		parts += "<span class='redtext'>The Syndicate failed their incursion!</span>"
+
+	return "<div class='panel redborder'>[parts.Join("<br>")]</div>"
+
+/datum/team/incursion/proc/add_objective(datum/objective/O, needs_target = FALSE)
+	O.team = src
+	if(needs_target)
+		O.find_target(dupe_search_range = list(src))
+	O.update_explanation_text()
+	objectives += O
+	for(var/datum/mind/member in members)
+		log_objective(member, O.explanation_text)
+
+/datum/team/incursion/proc/forge_team_objectives()
+	objectives = list()
+	var/is_hijacker = prob(20)
+	for(var/i = 1 to max(1, CONFIG_GET(number/incursion_objective_amount)))
+		forge_single_objective(CLAMP((4 + !is_hijacker)-i, 1, 3))	//Hijack = 3, 2, 1, 1 no hijack = 3, 3, 2, 1
```

Comment:
If they have hijack they do not get any hard objectives (needs to be 5 instead of 4 because of the -i and starting from 1)